### PR TITLE
feat: more changes to the CRD status

### PR DESCRIPTION
**Pull request by nbalacha**
The LVMCluster and LVMVoluemGroupNodeStatus will now include information about the devices being used by the configured volumegroups.

### CODE REVIEW

**Review by leelavg — APPROVED** [review]
/lgtm

**Review by jmolmo — APPROVED** [review]
Maybe we may need a device status in the future

File: pkg/vgmanager/vgmanager_controller.go [hunk 1]
```diff
@@ -158,6 +158,12 @@ func (r *VGReconciler) reconcile(ctx context.Context, req ctrl.Request) (ctrl.Re
 	}
 
 	_, found := deviceClassMap[volumeGroup.Name]
+	if found {
+		volGrpHostInfo, err := GetVolumeGroup(r.executor, volumeGroup.Name)
+		if err == nil {
```
Comment (sp98) [inline pkg/vgmanager/vgmanager_controller.go:163]: why not handle errors?

File: pkg/vgmanager/vgmanager_controller.go [hunk 2]
```diff
@@ -217,6 +223,11 @@ func (r *VGReconciler) reconcile(ctx context.Context, req ctrl.Request) (ctrl.Re
 		}
 	}
 
+	volGrpHostInfo, err := GetVolumeGroup(r.executor, volumeGroup.Name)
+	if err == nil {
```
Comment (sp98) [inline pkg/vgmanager/vgmanager_controller.go:227]: why not handle errors?

**Review by sp98 — CHANGES_REQUESTED** [review]

**Review by sp98 — APPROVED** [review]

File: pkg/vgmanager/vgmanager_controller.go (hunk 2 above)
Comment (nbalacha) [reply pkg/vgmanager/vgmanager_controller.go:229]: done

File: pkg/vgmanager/vgmanager_controller.go (hunk 1 above)
Comment (nbalacha) [reply pkg/vgmanager/vgmanager_controller.go]: done

**Pull request merged**